packages/interactivity-router/src/head.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Helper to update only the necessary tags in the head.
+ *
+ * @async
+ * @param {Array} newHead The head elements of the new page.
+ *
+ * @return {void}
```

Comment:
If there is no return value, you can skip `@return` line.

Reply:
Changed in [this commit](https://github.com/WordPress/gutenberg/pull/59707/commits/743926f0160e985a4641c3b158f64700e8c656ea).